service-front/module/Application/view/application/pages/what_is_vouching.twig

```diff
@@ -0,0 +1,63 @@
+{% extends "layout/layout" %}
+
+{% block beforeMain %}
+    {% include 'layout/id_check_banner.twig' with details_data %}
+{% endblock %}
+
+{% block content %}
+    <form method="POST" class="govuk-!-width-two-thirds">
+        <h1 class="govuk-heading-xl">{% block title %}What is Vouching?{% endblock %}</h1>
```

Comment:
Non-form elements inside forms can get weird for assistive tech, I'd move the `<form>` to wrap just the form-group with the radios at line 41.